packages/jaeger-ui/src/components/TracePage/TracePageSearchBar.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react';
+import { Button, Input } from 'antd';
+
+import * as markers from './TracePageHeader.markers';
```

Comment:
Please set the tests up so each component only uses it's own markers, in a file named after the component, e.g. `TracePageSearchBar.markers.js`.